### DISCUSSION TITLE
Partially qualified object names in ER Diagrams

### DISCUSSION THREAD

**Discussion by vince-vanh** [dbeaver/dbeaver#34212] [Q&A]
In ER diagrams, Is it possible to use the naming convention: schema_name.table_name? It seems the only 2 options are controlled by setting "fully qualified names" to true or false which results in names that are just the object name or the db name, schema name, AND object name.
e.g.
1. my_database.my_schema.my_table
or
2. my_table

The most helpful option in most cases is simply:
my_schema.my_table

**Comment by serjiokov** [comment]
Seems it might be more granularly configured like 
[database].[schema].[table]
or [schema].[table]
or [table]
